inst/resources/activate.R

```diff
@@ -69,6 +69,25 @@ local({
 
   })
 
+  check_project_synchronization <- function() {
+
+    capture.output({status <- renv::status()})
```

Comment:
You can use `quietly()` here:

https://github.com/rstudio/renv/blob/18d1169f04370d37021fce7dbc188bd256ecde22/R/utils.R#L294-L308